Entities/Characters/Builder/BuilderAnim.as

```diff
@@ -127,14 +127,17 @@ void onTick(CSprite@ this)
 		{
 			this.SetAnimation("crouch");
 		}
-		else if (action2 || ((this.isAnimation("strike") || this.isAnimation("chop")) && !this.isAnimationEnded()))
+		else if (action2 || ((this.isAnimation("strike") || this.isAnimation("strike_fast") || this.isAnimation("chop") || this.isAnimation("chop_fast")) && !this.isAnimationEnded()))
 		{
 			string attack_anim = blob.get_string("prev_attack_anim");
 			HitData@ hitdata;
 			if (blob.get("hitdata", @hitdata))
 			{
 				bool hitting_wood = false;
 				bool hitting_stone = false;
+
+				// faster hitting against player-built blocks: tiles/backwall/doors, platforms, ladders and spikes
+				bool hitting_built = false;
```

Comment:
`hitting_built` is an unclear variable name to me tbh, what about `hitting_structure`? Remove the comment (or put it behind the variable declaration) and put it together with the other variable declarations.

Reply:
resolved